homeassistant/components/wirelesstag/__init__.py

```diff
@@ -122,65 +144,38 @@ def install_push_notifications(self, binary_sensors):
                 )
             else:
                 _LOGGER.info(
-                    "Installed push notifications for all\
-                             tags in %s.",
-                    mac,
+                    "Installed push notifications for all tags in %s.", mac,
                 )
 
-    @property
-    def local_base_url(self):
-        """Define base url of hass in local network."""
-        if self._local_base_url is None:
-            self._local_base_url = f"http://{util.get_local_ip()}"
-
-            port = self.hass.config.api.port
-            if port is not None:
-                self._local_base_url += f":{port}"
-        return self._local_base_url
-
-    @property
-    def update_callback_url(self):
-        """Return url for local push notifications(update event)."""
-        return f"{self.local_base_url}/api/events/wirelesstag_update_tags"
-
-    @property
-    def binary_event_callback_url(self):
-        """Return url for local push notifications(binary event)."""
-        return f"{self.local_base_url}/api/events/wirelesstag_binary_event"
-
-    def handle_update_tags_event(self, event):
+    async def handle_tag_update(self, data):
         """Handle push event from wireless tag manager."""
-        _LOGGER.info("push notification for update arrived: %s", event)
+        _LOGGER.info("Push notification for update arrived: %s", data)
         try:
-            tag_id = event.data.get("id")
-            mac = event.data.get("mac")
-            dispatcher_send(self.hass, SIGNAL_TAG_UPDATE.format(tag_id, mac), event)
+            tag_id = data.get("id")
+            mac = data.get("mac")
+            async_dispatcher_send(
+                self.hass, SIGNAL_TAG_UPDATE.format(tag_id, mac), data
+            )
         except Exception as ex:  # pylint: disable=broad-except
```

Comment:
Please remove this generic exception. What can cause it to do this ?

Reply:
I'll remove it. I don't know what can cause it to except. Maybe now, nothing. I didn't author the original code and I was attempting minimal changes :)